http/vibe/http/client.d

```diff
@@ -156,14 +156,10 @@ private void httpRequesterDg(scope HTTPClientRequest req, in URL url, in HTTPCli
 	else
 		req.headers["Host"] = hoststr;
 
-	if ("authorization" !in req.headers && url.username != "") {
-		import std.base64;
-		string pwstr = url.username ~ ":" ~ url.password;
-		req.headers["Authorization"] = "Basic " ~
-			cast(string)Base64.encode(cast(ubyte[])pwstr);
-	}
+	if ("authorization" !in req.headers && url.username != "")
+		req.addBasicAuth(url.username, url.password);
 
-	if (requester) requester(req);
+	if (requester) () @trusted{ requester(req); } ();
```

Comment:
There's no point in using `@safe` if we trust a random external delegate.

Reply:
In fact, if look carefully, initially all of this code block was unsafe. Now only one line of it marked as trusted.

Reply:
In other words, we previously trusted him too and nothing changed.